### PR TITLE
Adding transparency data walk post and _config_dev.yml

### PR DESCRIPTION
**Pull request by rbshaffer**
_config_dev.yml is just a config file with default URLs deleted. Useful for local jekyll testing, for those who are interested.

### CODE REVIEW

**Comment by rbshaffer** [comment]
Updating my own personal post, and fixing an issue in _config.yml (dates still refer to 2016 rather than 2017).

File: _config_dev.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: Data Science for Social Good 2016
```
Comment (valentina-s) [inline _config_dev.yml]: 2016 ->2017

File: _posts/2017-06-16-transparency-data-walk.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ 
+The walk raised several questions about how data is made available and how the community responds to data.
+
+<img src="/assets/images/1_ppt1.PNG" style="width: 750px;"/>
+<img src="/assets/images/1_ppt2.PNG" style="width: 750px;"/>
+<img src="/assets/images/1_ppt3.PNG" style="width: 750px;"/>
+ 
+## Conclusion
+Ultimately, our short data walk from the streets of Seattle’s University district to the waterfront revealed the massive amounts of data that surround us. More than that, the walk revealed the myriad ways in which we can interpret this data. While our group focused on data transparency and accessibility, other groups focused on different aspects of data exploration. Our ability to understand data is driven by the lenses that we view data through; the data that we see, while the same as what others see, may be interpreted vastly differently. As we begin our projects, considerations to such lenses will be essential, both to maximize our chances of an ethical and successful project as well as to control the negative effects that bias plays in all data science.
```
Comment (valentina-s) [inline _posts/2017-06-16-transparency-data-walk.md]: this data -> these data

File: _posts/2017-06-16-shaffer.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+layout: post
+title:  "Introducing DSSG Fellow Robert Shaffer"
+date:   2017-06-17
+author: Robert Shaffer
+---
+
+![Robert]({{ site.url }}/assets/images/Robert.jpg){:width="200px" align="right" style="PADDING-LEFT: 5px; PADDING-BOTTOM: 5px;"}
+I am a PhD candidate in Government at the University of Texas at Austin.
+Most of my work involves applying statistical NLP tools to study formal
+legal texs (e.g. national constitutions or ordinary legislation). I am
```
Comment (valentina-s) [inline _posts/2017-06-16-shaffer.md]: texs -> texts

**Comment by valentina-s** [comment]
I added comments regarding fixing a few typos.

**Comment by rbshaffer** [comment]
Okay, just updated those!

**Comment by valentina-s** [comment]
Thanks!

**Pull request merged**